tanner/emulators/php_object_injection.py

```diff
@@ -0,0 +1,53 @@
+import aiohttp
+import asyncio
+import logging
+
+from tanner import config
+from tanner.utils import patterns
+
+
+class PHPObjectInjection:
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.php_object_injection')
+
+    async def get_injection_result(self, code):
+        object_injection_result = None
+
+        vul_code = "<?php " \
```

Comment:
I think we can make it better like [this](https://www.netsparker.com/blog/web-security/untrusted-data-unserialize-php/) with more intuitive class. Also, keep it simple, like just pass the input and append it to a file in `/tmp` e.g 
`system("$log >> /tmp/logs")` with $log contains contains $insert, date, time etc. This is a simple example, you can think of others.

Reply:
Actually, its like this because of `phpox`. `system` function in it has to be passed two params and it returns the result in the array, rest part is very straight forward.

Reply:
What is the problem? I don't understand.

Reply:
There isn't any problem, im just clearing up as you said its a bit complicated.

Reply:
Used a kind of similar idea to make it more intuitive :)